tools/ci/actions/changes.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+#**************************************************************************
+#*                                                                        *
+#*                                 OCaml                                  *
+#*                                                                        *
+#*                 David Allsopp, OCaml Labs, Cambridge.                  *
+#*                                                                        *
+#*   Copyright 2021 David Allsopp Ltd.                                    *
+#*                                                                        *
+#*   All rights reserved.  This file is distributed under the terms of    *
+#*   the GNU Lesser General Public License version 2.1, with the          *
+#*   special exception on linking described in the file LICENSE.          *
+#*                                                                        *
+#**************************************************************************
+
+#------------------------------------------------------------------------
+#This test checks that the Changes file has been modified by the pull
+#request. Most contributions should come with a message in the Changes
+#file, as described in our contributor documentation:
+#
+#  https://github.com/ocaml/ocaml/blob/trunk/CONTRIBUTING.md#changelog
+#
+#Some very minor changes (typo fixes for example) may not need
+#a Changes entry. In this case, you may explicitly disable this test by
+#adding the code word "No change entry needed" (on a single line) to
+#a commit message of the PR, or using the "no-change-entry-needed" label
+#on the github pull request.
+#------------------------------------------------------------------------
```

Comment:
The corresponding [Travis check](https://github.com/ocaml/ocaml/blob/d83e466615d030542faf048470cf45ef665794a0/tools/ci/travis/travis-ci.sh#L246-L265) is careful to print this explanation as a message, instead of having this as a comment. When the check fails, contributors see the logs, not the code of the action; printing the explanation as a message makes it clear to them what the test is trying to do, and what action to take to resolve the failure.

Could we use the same approach here?

Reply:
That's a mistake on my part!